extractor.go

```diff
@@ -404,6 +457,16 @@ func mapBlockID(bid types.BlockID) *codec.BlockID {
 	}
 }
 
+func mapProposer(val *types.Validator) *codec.Validator {
+	nPK := &codec.PublicKey{}
+
+	return &codec.Validator{
+		Address:          val.Address,
+		PubKey:           nPK,
+		ProposerPriority: 0,
```

Comment:
I have a feeling that we dont even need this `ProposerPriority` field, see my comment in the code of `mapValidatorUpdate` function on master branch.